BatchPreAnalysis_config.ini

```diff
@@ -0,0 +1,14 @@
+[SOFTWARE]
```

Comment:
same as BatchPreAnalysis.py -> remove from PR

Reply:
This came up as the target branch was set to "master" instead of "automation-robustness" → branch is set to "automation-robustness" now.